src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/DraftReferralServiceUnitTest.kt

```diff
@@ -357,7 +358,7 @@ class DraftReferralServiceUnitTest {
     @Test
     fun `updating selected service category with same set doesn't remove complexity level for same service categories`() {
 
-      val complexityLevel1 = ComplexityLevel(UUID.randomUUID(), "title", "description")
+      val complexityLevel1 = ComplexityLevel(UUID.randomUUID(), "title", "description", Complexity.LOW)
```

Comment:
💭 After this pull request, we could deprecate the `title` field and assume it from the enum value (it's always the same, after all)

Reply:
Yes true @sldblog , I will raise a ticket for that in the backlog